packages/mobile/src/account/saga.ts

```diff
@@ -97,6 +99,10 @@ function* migrateAccountToProperBip39() {
   }
 }
 
+function* clearStoredAccount({ account }: ClearStoredAccountAction) {
```

Comment:
Consider calling this `clearStoredAccountWrapper` or `clearStoredAccountSaga` to differentiate from the action and I'd recommend adding some error handling here. That's a typical pattern we have for our sagas. It's better to have them catch errors and show the error banner instead of letting those errors propagate, potentially killing the whole app.
Here's an example:
https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/send/saga.ts#L171

Reply:
I did the error handling that way and added a few more things, but I'm not sure about the banner thing. Most of the things we're doing here are just removing values which have their own error handling, so I think it's unlikely that we get an error here, but if we do get an error it would be either in `ValoraAnalytics` or when signing out of Firebase. When those run the account is already deleted so I think it's fine to ignore them. Right now, the `Settings` screen fires this saga and just navigates to the onboarding screen, ignoring any possible error. I'm not sure that's the best behavior, but if analytics or firebase fail the consequences don't seem very bad compared to the alternative.
What do you think?